pkg/controller/certificaterequests/venafi/venafi_test.go

```diff
@@ -146,6 +146,10 @@ func TestSign(t *testing.T) {
 		}),
 	)
 
+	tppCRWithCustomFields := gen.CertificateRequestFrom(tppCR, gen.SetAnnotations(map[string]string{"venafi.cert-manager.io/custom-fields": `[{"name": "cert-manager-test", "value": "test ok"}]`}))
+
+	tppCRWithInvalidCustomFields := gen.CertificateRequestFrom(tppCR, gen.SetAnnotations(map[string]string{"venafi.cert-manager.io/custom-fields": `[{"name": cert-manager-test}]`}))
```

Comment:
use the consts here too :)

Reply:
On one hand I agree on one I doubt... I would love for some test to fail if the const changes somehow as if that happens some production service will break...